packages/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -34,6 +34,8 @@ export interface WizardStep {
   hideCancelButton?: boolean;
   /** (Unused if footer is controlled) True to hide the Back button */
   hideBackButton?: boolean;
+  /** Force to disable the step in the navigation.*/
+  isDisabled?: boolean;
```

Comment:
I think the main thing for me is whether we can just use the `canJumpTo` prop for this new logic instead of introducing a prop with the same use-case. What conflict did you notice when modifying `canJumpTo`?

Reply:
I added this comment on the PR: 
` modify the prop canJumpTo creates conflict with the progressive wizard, so I added a new prop called isDisabled.`

This is because canJumpTo is calculated following a condition.